modules/performance.nix

```diff
@@ -60,6 +60,12 @@ in
       luaLib = lib.mkEnableOption "luaLib" // {
         description = "Whether to byte compile lua library.";
       };
+      excludePlugins = lib.mkOption {
+        type = with types; listOf (either str package);
+        default = [ ];
+        example = [ "faster.nvim" ];
```

Comment:
Would a `literalExpression` example that demonstrates using a `pkgs.vimPlugins.*` package, alongside the string, be useful?